Gemfile

```diff
@@ -4,6 +4,8 @@ Encoding.default_internal = Encoding::UTF_8
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}" }
 
+gem "vectory", github: "metanorma/vectory", branch: "integrate-metanorma-utils"
```

Comment:
Is this only added to pass the tests? When we merge we want to remove this line since it’s already in the gemspec.

Reply:
Yes, of course 👍